Fix appendix instruction with native asset as fee

yrong opened the pull request.

alistair-singh commented on web/packages/api/src/toEthereum_v2.ts:
```diff
@@ -346,7 +347,8 @@ export async function getDeliveryFee(
             assetHubExecutionFeeDOT +
-            returnToSenderExecutionFeeDOT +
+            // Increase the returnToSenderExecutionFeeDOT by 5x in an edge case to ensure the token is sent back, mainly for the Mythos parachain
+            returnToSenderExecutionFeeDOT * 5n +
```
Multiply by 5 twice. Once here.

yrong commented:
@Alistair Singh Unfortunately, option 1 doesn’t work. As you may have noticed in previous commit, I added another `initiateTeleport` to pre-fund the sovereign account of AH on Mythos and attempted to withdraw from it to cover the fees. However, there’s a `ClearOrigin` right after the `ReserveAssetDeposited(SBMyth)` instruction, which causes it to fail.

So I switched to option 2. Tested with an intentionally triggered error using [this extrinsic](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Fmythos.ibp.network#/extrinsics/decode/0x1f0d04010100a10f0408000000138dc13dde8fcf51aa02020907040300ba41ddf06b7ffd89d1267b5a93bfef2424eb20030002286bee0204000000040c16080f0101010100a5340004010000038030464f000e0008010000038030464f02020907040300ba41ddf06b7ffd89d1267b5a93bfef2424eb20030002286bee010100a5340c13010000038030464f000d01020800010300302f0b71b8ad3cf6dd90adb668e49b2168d652fd2c8403a347664da006f2ffcbed4481105c64f5275b6bc4174bd9f6f3ea8b5a4f8810010102020907040300ba41ddf06b7ffd89d1267b5a93bfef2424eb20030002010907040c1300010300ba41ddf06b7ffd89d1267b5a93bfef2424eb20030004000d01020400010300302f0b71b8ad3cf6dd90adb668e49b2168d652fd2c8403a347664da006f2ffcbed4481105c64f5275b6bc4174bd9f6f3ea8b5a4f882c8403a347664da006f2ffcbed4481105c64f5275b6bc4174bd9f6f3ea8b5a4f8800) 

From the [event](https://assethub-polkadot.subscan.io/xcm_message/polkadot-5de9b2372e48ab7aba69ee4e2670f848e963a90e), it demonstrates returning assets to the source chain using `setAppendix` worked as expected.
